docs/import-from-whitehall.md

```diff
@@ -0,0 +1,110 @@
+# Importing documents from Whitehall to Content Publisher
+
+There are two ways to import documents from Whitehall:
+- a single document
+- documents of a specific type or subtypes from a single publishing organisation
+
+## Running an import
+
+### Single document import
+
+This will import a single document and all its editions from Whitehall into Content Publisher.
+
+```
+rake import:whitehall_document[1234]
+```
+
+Where `1234` is the Whitehall document ID.  After running the rake task, a URL will be printed to the console which allows the status of import to be examined.
+
+Note that the ID in Whitehall's internal URLs is the edition number (e.g. for `/government/admin/publications/1034451` the edition is `1034451`).  The document ID can be obtained on a Rails console, using:
+
+```
+Edition.find(1034451).document.id
+```
```

Comment:
Might be good to reference that you have to be using a Whitehall Rails console and it needs to be in the environment. We could also give a command that takes the whole URL rather than giving people instructions of what to pull out the URL. For example:

```
irb(main):005:0> URI.parse("https://whitehall-admin.integration.publishing.service.gov.uk/government/admin/publications/1038279").path.split("/")[4].then { |id| Edition.find(id).document_id }
=> 412144
```

Reply:
Updated the text above to say it must be a Whitehall Rails console.  Also changed the code to parse a URL.